Project.toml

```diff
@@ -15,12 +15,14 @@ JSONFBCModels = "475c1105-d6ed-49c1-9b32-c11adca6d3e8"
 JuMP = "4076af6c-e467-56ae-b986-b466b2749572"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
+Tulip = "6dd1b50a-3aae-11e9-10b5-ef983d2400fa"
```

Comment:
tulip here should only be in extras, the package itself doesn't depend on it.

OTOH tests depend on it AND the docs depends on it.